websites/A/AnimeFLV/dist/metadata.json

```diff
@@ -16,7 +16,7 @@
   },
   "url": [
     "www3.animeflv.net",
-    "animeflv.ru"
+    "animeflv.ac"
   ],
   "version": "1.1.8",
   "logo": "https://i.ibb.co/k1x0mXZ/descarga.png",
```

Comment:
could you upload logo and thumbnail to imgur thanks heh

Reply:
Done!